workflows/iuf/stages.yaml

```diff
@@ -118,15 +118,7 @@ stages:
   - name: management-nodes-rollout
     type: global
     operations:
-      - name: info-to-read
-        static-parameters: {} # any parameters that will be supplied statically to this operation.
-      - name: verify-images-and-configuration
-        static-parameters: {} # any parameters that will be supplied statically to this operation.
-      - name: management-storage-nodes-rollout
-        static-parameters: {} # any parameters that will be supplied statically to this operation.
-      - name: management-two-master-nodes-rollout
-        static-parameters: {} # any parameters that will be supplied statically to this operation.
-      - name: management-worker-nodes-rollout
+      - name: management-nodes-rollout
```

Comment:
With the changes of refactoring, there is no operation `management-nodes-rollout`. All the refactored operations should be called individually.

Reply:
This is a good catch but this stage works differently than the others. The IUF backend handles getting the correct workflow for NCN type based on inputs to --limit-management-rollout parameter. The code that does this is [here](https://github.com/Cray-HPE/cray-nls/blob/0597b3fb49c966d260fac173b82bd224a3467af3/src/api/services/iuf/sessions_workflow_gen.go#L636).

Reply:
Ahh, This is new information to me. Good to know that. Thanks.